src/api/system_api.js

```diff
@@ -14,6 +14,31 @@ module.exports = {
     id: 'system_api',
 
     methods: {
+        get_system_status: {
+            doc: 'Query for the status of a system',
+            method: 'GET',
+            reply: {
+                type: 'object',
+                required: ['state', 'last_state_change'],
+                properties: {
+                    state: {
+                        type: 'string',
+                        enum: [
+                            'DOES_NOT_EXIST',
+                            'INITIALIZING',
+                            'COULD_NOT_INITIALIZE',
+                            'READY',
+                        ]
+                    },
+                    last_state_change: { idate: true }
```

Comment:
why do we need it ?

Reply:
This will enable us to take action in case of long phases, for example we could find out that a system  is stuck on initializing state in the operator and we might decide to undo the creation